streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamThreadTest.java

```diff
@@ -1936,7 +1934,7 @@ public void process(final Object key, final Object value) {
         clientSupplier.consumer.addRecord(new ConsumerRecord<>(
             topic1,
             1,
-            0L,
+            100L,
```

Comment:
nit: can we pick a different value to disambiguate (wall-clock time is set to `100L` above)?

Reply:
Ack.